tools/hitmap/hitmap.xml

```diff
@@ -0,0 +1,297 @@
+
+<tool id="hitmap" name="HiT-MaP" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" license="MIT">
```

Comment:
the license refers, I think, to code that you ship with this tool not the license of the tool

Can you please also include a profile="20.05"